src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -203,33 +205,24 @@ public static String getIndividualFullName(Entity entity) {
         return entity.getName().getFullNameNoTitle();
     }
 
-    public static String getIndividualPreferredHearingChannel(String hearingType,
-                                                                        HearingSubtype hearingSubtype,
+    public static HearingChannel getIndividualPreferredHearingChannel(HearingSubtype hearingSubtype,
                                                                         HearingOptions hearingOptions) {
-        if (eitherNull(hearingType, hearingSubtype)) {
+        if (isNull(hearingSubtype) || isNull(hearingOptions)) {
             return null;
         }
 
         HearingChannel preferredHearingChannel =
-            shouldPreferNotAttendingHearingChannel(hearingType, hearingOptions) ? NOT_ATTENDING
-            : isYes(hearingSubtype.getWantsHearingTypeFaceToFace()) ? FACE_TO_FACE
-            : shouldPreferVideoHearingChannel(hearingSubtype) ? VIDEO
-            : shouldPreferTelephoneHearingChannel(hearingSubtype) ? TELEPHONE
-            : null;
+            hearingOptions.isWantsToAttendHearing()
+                ? isYes(hearingSubtype.getWantsHearingTypeFaceToFace()) ? FACE_TO_FACE
+                : shouldPreferVideoHearingChannel(hearingSubtype) ? VIDEO
+                : shouldPreferTelephoneHearingChannel(hearingSubtype) ? TELEPHONE
+                : null : NOT_ATTENDING;
```

Comment:
This is very confusing, can you refactor this into if statements, intellij has a feature that automatically does this

Reply:
Good shout have now converted it to a more cleaner way